bfvmm/include/memory_manager/arch/x64/unique_map.h

```diff
@@ -82,16 +68,14 @@ class unique_map_ptr;
 ///
 template<class T>
 auto make_unique_map(
-    typename unique_map_ptr<T>::pointer phys,
-    ::x64::memory_attr::attr_type attr = ::x64::memory_attr::rw_wb)
```

Comment:
Is there another way to set the attributes somewhere? The extended apis requires UC for the xAPIC. The comments in this file need to be updated to reflect the new params as well

Reply:
No, not at the moment, I will fix that (might take a bit so I will close the PR for now). I will likely only add support for UC and WB to keep it simple. The previous implementation was far too complicated and caused problems